web/yo/app/scripts/services/stringutils.js

```diff
@@ -1091,8 +1091,8 @@ angular.module('oncokbApp')
                 if (model.get(e)) {
                     if (model.get(e).type === 'Map') {
                         reviewObj[e] = getReview(model[e]);
-                    } else {
-                        reviewObj[e] = getString(model.get(e));
+                    } else if(['currentReviewer', 'review', 'action', 'rollback', 'loading'].indexOf(e) === -1) {
```

Comment:
This will not prevent similar issue happen in the future, right? Shouldn't we just use _.isString(model.get(e))?